curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f805f813eaf2049bc2ceea.md

```diff
@@ -26,25 +26,29 @@ assert.equal(document.querySelectorAll('label input')?.length, 4);
 You should add the first `input` after the `label` text `Enter Your First Name:`, and include a space after the colon.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.[0]?.innerHTML.trim(), 'Enter Your First Name: <input>');
+const query = /^Enter Your First Name:\s+<input>/
+assert.equal(query.test(document.querySelectorAll('label')?.[0]?.innerHTML.trim()),true);
```

Comment:
```suggestion
assert.match(document.querySelectorAll('label')?.[0]?.innerHTML.trim(), query);
```

Reply:
Woah. I didn't even know that method was a thing. All I relatively know was the Chai assertion library.